source/config-file/protect-connection-strings.txt

```diff
@@ -0,0 +1,71 @@
+.. _compass-protect-connection-strings:
+.. _hide-connection-string-credentials:
+
+==========================================
+Hide Credentials in Your Connection String 
+==========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can use the ``protectConnectionStrings`` option to hide credentials when 
```

Comment:
[suggestion]

Consider rephrasing this opening sentence we say the goal of this procedure first, followed by how to achieve it. Something like:

"When you paste a connection string into the |compass-short| connection form, by default, |compass-short| shows credentials in plaintext. To hide credentials in the connection string, use the ``protectConnectionStrings`` option."

Reply:
Thanks for the suggestion in motivating the task! Updated.